domain/model/state/controllerstate.go

```diff
@@ -1260,26 +1295,26 @@ LEFT JOIN model_namespace mn ON m.uuid = mn.model_uuid
 WHERE m.uuid = $dbUUID.uuid
 `, dbModelNamespace{}, mUUID)
 	if err != nil {
-		return "", errors.Annotate(err, "preparing select model namespace statement")
+		return "", errors.Capture(err)
 	}
 
 	var namespace sql.NullString
 	err = db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
 		var result dbModelNamespace
 		if err := tx.Query(ctx, stmt, mUUID).Get(&result); errors.Is(err, sqlair.ErrNoRows) {
-			return fmt.Errorf("%w for id %q", modelerrors.NotFound, id)
+			return errors.Errorf("%w for id %q", modelerrors.NotFound, id)
```

Comment:
Can we fix it by using Add()?

Reply:
Possibly. It is about moving the package not fixing what is existing at the moment.

Reply:
Done & Done